README.md

```diff
@@ -127,16 +128,22 @@ The following is a list of static resources.
 - [azure_event_hub_namespace](docs/resources/azure_event_hub_namespace.md)
 - [azure_graph_user](docs/resources/azure_graph_user.md)
 - [azure_graph_users](docs/resources/azure_graph_users.md)
+- [azure_generic_resource](docs/resources/azure_generic_resource.md)
+- [azure_generic_resources](docs/resources/azure_generic_resources.md)
+- [azure_graph_generic_resource](docs/resources/azure_graph_generic_resource.md)
+- [azure_graph_generic_resources](docs/resources/azure_graph_generic_resources.md)
```

Comment:
we will not add this to doc.. coz these Objects(which are named as resources) are used only for inheritance and creating specific targeted resources only and not pointing to any external Azure resources.. So customers cannot use it like an inspec resource is what